lib/node/types.ml

```diff
@@ -0,0 +1,26 @@
+open Lwt_unix
+open Common
+
+type failure_detector_config = {
```

Comment:
Should it belong to `failure_detector`?

Reply:
Cyclic dependency issue. The types module exists so that we don't have modules depending on one another 😅